ArduCopter/mode_guided.cpp

```diff
@@ -739,6 +770,11 @@ void ModeGuided::posvelaccel_control_run()
         return;
     }
 
+    // if set and no pilot input for 2 sec weathervane copter into wind
+    if (allows_weathervaning()) {
```

Comment:
I wonder if we want weathervaning available for all the guided mode controllers.  My feeling is that perhaps it should only be available for the wp control (e.g. what happens when the user clicks on "Fly Here" in the GCS).

Reply:
Whilst I agree that it is less likely for weathervaning to be used in posvel its not impossible to image users wanting it.  I would much rather we leave it in so that it is more consistant across the guided controllers.  The differnce in guided controllers is already pretty confusing without adding to the differences.